cmake/std/atdm/README.md

```diff
@@ -254,6 +256,55 @@ $ srun ./checkin-test-sems.sh intel-opt-openmp \
   --local-do-all
 ```
 
+### chama/serrano
+
+Once logged on to `chama`  or `serrano`, one can
+directly configure and build on the login node (being careful not to overload
+the node).  But to run the tests, one must run on the compute nodes using the
+`srun` command.  For example, to configure, build and run the tests for say
+`MueuLu` on `serrano` or `chama`, (after cloning Trilinos on the `develop` branch) one
+would do:
+
+
+```
+$ cd <some_build_dir>/
+
+$ source $TRILINOS_DIR/cmake/std/atdm/load-env.sh intel-opt-openmp
+
+$ cmake \
+  -DTrilinos_CONFIGURE_OPTIONS_FILE:STRING=cmake/std/atdm/ATDMDevEnv.cmake \
+  -DTrilinos_ENABLE_TESTS=ON -DTrilinos_ENABLE_MueLu=ON \
+  $TRILINOS_DIR
+
+$ make -j16
+
+$ srun -N 1 --time=600 --account=<YOUR_WCID> -J $JOB_NAME  -e output.error ctest -j16
```

Comment:
@fryeguy52,

Can we take out `-e output.error`?   I found that just hides the error output and no one ever notices.

Reply:
Note that the file `ctest/cmake/drivers/atdm/toss3/local-driver.sh` does not have `-e output.error` so we should not need it here either.